### DISCUSSION TITLE
Correct place for raising documentation improvements

### DISCUSSION THREAD

**Discussion by lmac-1** [amannn/next-intl#652] [General]
Hello there! 

First, I just wanted to say thank you for all of your hard work to implement server component support. Your changes have come just in time for me! 😎

I'm going through the documentation for the first time to add `next-intl` to my project. Where is the right place to raise any suggestions/improvements/feedback for the docs? I looked at Issues but only saw templates for raising a bug or feature request.

I just spotted a few points where I got a bit confused and a couple of small notes might help others in the same situation. 

Have a great day!
Lucy

**Comment by lmac-1** [comment]
By the way, when I clicked the button "Question? Give us feedback." link from the docs, it took me to the New Issue page and so I wasn't sure what to choose

![image](https://github.com/amannn/next-intl/assets/64803272/977abb9e-8770-4d8f-a6de-115232d04b03)

I'm not sure if this is supposed to link somewhere else as I can see the link url is actually https://github.com/amannn/next-intl/issues/new?title=Feedback%20for%20%E2%80%9CNext.js%20App%20Router%20Internationalization%20(i18n)%E2%80%9D&labels=feedback
but it redirects me to https://github.com/amannn/next-intl/issues/new/choose

**Reply by amannn** [reply]
That's interesting, thanks for the report! For me the link works, but that might be because I have admin rights on the repo.

Maybe a template needs to be chosen, can you try this link?

https://github.com/amannn/next-intl/issues/new?template=update_docs.yml&title=%5BDocs%5D%3A%20A+test

**Reply by lmac-1** [reply]
That link works for me :) 

![image](https://github.com/amannn/next-intl/assets/64803272/8d01b4d5-3f33-481b-aec4-50b84eb2c895)

**Reply by amannn** [reply]
Great, thanks for trying it out! I've fixed the feedback link in https://github.com/amannn/next-intl/pull/659.

**Comment by amannn** [comment]
Hello Lucy,

so nice to hear from you and thank you for the kind words!

I'd be more than happy to learn about your feedback, the docs are [a very important part of `nex-intl`](https://github.com/amannn/next-intl/blob/main/CONTRIBUTORS.md). I've just added [a new issue template for docs](https://github.com/amannn/next-intl/issues/new/choose)—feel free to use that!

**Reply by lmac-1** [reply]
Thanks a lot! :)
